android/app/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "controlinator.finance.finance_controlinator_mobile"
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion 18
```

Comment:
Added why the version was upgraded in FinanceControlidationDocs/identity/v1/Identity Architecture